dhis-2/dhis-api/src/main/java/org/hisp/dhis/trackedentity/TrackedEntityInstanceQueryParams.java

```diff
@@ -377,6 +383,11 @@ public void handleCurrentUserSelectionMode()
         }
     }
     
+    public boolean hasTrackedEntityInstances()
+    {
+        return this.trackedEntityInstanceUids != null && !this.trackedEntityInstanceUids.isEmpty();
```

Comment:
Might be more readable:
```suggestion
        return CollectionUtils.isNotEmpty( this.trackedEntityInstanceUids );
```